Riot/Assets/en.lproj/Vector.strings

```diff
@@ -798,6 +798,7 @@ Tap the + to start adding people.";
 "settings_labs_enable_new_client_info_feature" = "Record the client name, version, and url to recognise sessions more easily in session manager";
 "settings_labs_enable_new_app_layout" = "New Application Layout";
 "settings_labs_enable_wysiwyg_composer" = "Try out the rich text editor (plain text mode coming soon)";
+"settings_labs_enable_voice_broadcast" = "Voice broadcast (under active development). We currently only detect voice broadcast in the room timeline, this is not possible to send or listen an actual voice broadcast.";
```

Comment:
I would remove the point at the end of the string. The other Labs labels don't have point